src/pages/ForgotPasswordPage/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import ROUTES from '../../utils/routes';
+import ForgotPasswordForm from '../../containers/ResetPasswordForm';
+
+class ResetPasswordPage extends Component {
+  render() {
+    return (
+      <>
```

Comment:
Hi @mutaimwiti,
For some reason I find line `4` very weird because we are importing `ForgotPasswordForm` from `ResetPasswordForm` or they are just similar components that you are reusing?

Reply:
@SrMoffat 
1. You need to access the `/forgot-password` route for you to access the UI.
2. My bad. I did it accidentally while creating the skeleton.